inotify.go

```diff
@@ -278,27 +275,6 @@ func (w *Watcher) readEvents() {
 	}
 }
 
-// Certain types of events can be "ignored" and not sent over the Events
-// channel. Such as events marked ignore by the kernel, or MODIFY events
-// against files that do not exist.
-func (e *Event) ignoreLinux(mask uint32) bool {
-	// Ignore anything the inotify API says to ignore
-	if mask&syscall.IN_IGNORED == syscall.IN_IGNORED {
```

Comment:
In general I'm in favour of this library passing as much information straight from inotify to our application as possible - dropping events at multiple places in the stack like this makes it harder to debug issues, IMO. I've got another ticket to add more detail to the events from fsnotify so we can know about all the flags.